core/src/queue.rs

```diff
@@ -646,7 +615,7 @@ mod tests {
             query_handle,
         ));
         let queue = Queue::from_config(Config {
-            transaction_time_to_live: Duration::from_millis(200),
+            transaction_time_to_live: Duration::from_millis(300),
```

Comment:
This change breaks this test for me, and I don't think this change can be right without changing the expected numbers of transactions collected

Reply:
Yes, i fixed this test in #4319